### PR TITLE
Update frontpage progress bar and fix some minor bugs

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

**Comment by Raemon** [comment]
cosmetic note: the TagFlagToggles should have a border when clicked so that they don't weirdly change size.

File: packages/lesswrong/components/tagging/EditTagPage.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const EditTagForm = ({tag, successCallback, cancelCallback}: {
     collection={Tags}
     documentId={tag._id}
     queryFragment={getFragment('TagEditFragment')}
-    mutationFragment={getFragment('TagEditFragment')}
+    mutationFragment={getFragment('TagFragment')}
```
Comment (Raemon) [inline packages/lesswrong/components/tagging/EditTagPage.tsx:17]: can you explain a bit about what's going on here and why the query and mutation fragments are different?

**Comment by Raemon** [comment]
(PR review done, just wanting a bit of clarity on the fragment)

File: packages/lesswrong/components/tagging/EditTagPage.tsx (hunk 1 above)
Comment (Discordius) [reply packages/lesswrong/components/tagging/EditTagPage.tsx:17]: They are almost always different. The query fragment determines what data is used to populate the form. The mutation fragment determines what data comes back after you submit the form. The first one usually includes lots of data for the revision and editor fields, and the second one are just the resulting data fields.

**Pull request merged**